### PR TITLE
Upgrade docu website to Node 18 & SvelteKit

### LINKED ISSUE #2315: Migrate to SvelteKit
**Description**

Migrate from Sapper to SvelteKit

**Reasons**

Sapper is no longer actively developed and SvelteKit is a much better experience

**Attachments**

Migration guide: https://kit.svelte.dev/docs/migrating

### PR DESCRIPTION
**Pull request by ndricimrr**
**Description**

Changes proposed in this pull request:

- Delete irrelevant files and sapper code
- Move to SvelteKit
- Update packages and remove unused or replace old dependencies. 
- ...

**Related issue(s)**

Fixes #2315

### CODE REVIEW

**Comment by ndricimrr** [comment]
script make - windows compatible

**Review by alexandra-simeonova — APPROVED** [review]
LGTM! 🎉 
I looked at the docu and tried adding a new test page, everything works: 
 
Screenshot 2023-06-21 at 15 27 37

File: website/docs/README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+    `npm install`
+
+2. Build Project.
+
+    `npm run build`
+
+3. Run Project: This command will serve Luigi Core on Port 4000 and the Client Microfrontends on Port 4001 on separate servers in parallel. To make sure both servers are running locally run them in separate Terminals
+
+    `npm run core` and ``npm run client`
```
Comment (ndricimrr) [inline website/docs/README.md:16]:
```suggestion
    `npm run core` and `npm run client`
```

**Review by JohannesDoberer — APPROVED** [review]
Good Job 👍

**Pull request merged**